pkgs/http_profile/lib/http_profile.dart

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:io';
+
+/// Records information about an HTTP request.
+final class HttpClientRequestProfile {
+  /// Determines whether HTTP profiling is enabled or not.
```

Comment:
[nit] Avoid phrases for getters. For booleans we often start the doc with "Whether".

```suggestion
  /// Whether HTTP profiling is enabled for the current zone's [HttpClient].
```

Reply:
I'm not sure if we want to pull the zone concept in here - I think that this is actually per isolate. But I stared the sentence with "Whether".